### ISSUE TITLE
Feature: Green squares is not accesible

### ISSUE THREAD

**Issue by bdougie** [open-sauced/landing-page#88]
### Type of feature

🍕 Feature

### Current behavior

reported [here](https://twitter.com/mislav/status/1603062302420668416). 

> The “We are more than green squares” graphic is visually cool, but is marked up by 4 distinct block elements. A screen reader would likely read that as 4 paragraphs. The graphic could be marked up by a single <p> containing spans to chop it up visually via CSS.

<img width="1361" alt="Screen Shot 2022-12-14 at 9 35 50 AM" src="https://user-images.githubusercontent.com/5713670/207667107-e1dd5f9b-07b2-4686-86a9-7c37db84b1be.png">

### Suggested solution

This image should convert to a custom CSS.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by SimranjitKaurMaan** [comment]
.take

**Comment by SimranjitKaurMaan** [comment]
can we modify the `svg` image to fit our purpose instead of custom css?

**Comment by shamimbinnur** [comment]
> can we modify the `svg` image to fit our purpose instead of custom CSS?

@SimranjitKaurMaan Why do you think it's necessary to modify the green-square SVG?

**Comment by 0-vortex** [comment]
> can we modify the `svg` image to fit our purpose instead of custom css?

No, the scope of the issue is to replace it with css as described in the suggested solution: 
<img width="352" alt="Screenshot 2023-01-14 at 18 00 24" src="https://user-images.githubusercontent.com/237133/212481478-15125c06-0133-4fdb-b060-244c460c689b.png">


If you don't know where to start with that, we can help if you ask questions!

**Comment by a0m0rajab** [comment]
Should this be closed as not planned due to having a different design right now?

**Comment by bdougie** [comment]
Still exist here https://opensauced.pizza/teams

<img width="1680" alt="Screen Shot 2023-07-26 at 6 59 48 AM" src="https://github.com/open-sauced/landing-page/assets/5713670/3453958c-c195-47cf-a302-ef515ed43883">

**Comment by dev-phantom** [comment]
> > can we modify the `svg` image to fit our purpose instead of custom css?
> 
> No, the scope of the issue is to replace it with css as described in the suggested solution: <img alt="Screenshot 2023-01-14 at 18 00 24" width="352" src="https://user-images.githubusercontent.com/237133/212481478-15125c06-0133-4fdb-b060-244c460c689b.png">
> 
> If you don't know where to start with that, we can help if you ask questions!

where do i start with this?

**Comment by a0m0rajab** [comment]
I did not have a look at the code, but based on the suggested solution, you might start by googling `SVG to CSS` and see how things go for you.

**Comment by CBID2** [comment]
.take

**Comment by dev-phantom** [comment]
> .take

Sorry to ask
Don't mean to sound rude

Please how do you intend to fix this?

**Comment by CBID2** [comment]
> > .take
> 
> 
> 
> Sorry to ask
> 
> Don't mean to sound rude
> 
> 
> 
> Please how do you intend to fix this?

Hi @dev-phantom! :) I'm going to convert the svg into CSS with a converter. Then, I'll lookup the colors' accessibie counterparts.

**Comment by CBID2** [comment]
> > > .take
> > 
> > 
> > Sorry to ask
> > Don't mean to sound rude
> > Please how do you intend to fix this?
> 
> Hi @dev-phantom! :) I'm going to convert the svg into CSS with a converter. Then, I'll lookup the colors' accessibie counterparts.

Hey @dev-phantom. I created the PR for this issue. I'd love to have your feedback on it. It's #164

**Comment by shamimbinnur** [comment]
Hey @CBID2, any updates on this issue?
I'm unsure why we're converting this image into CSS if screen readers can't interpret CSS elements. My suggestion is to leave the 'Green Square' as it is and simply add the alt text 'Green Square' for accessibility.

CC @bdougie

**Comment by CBID2** [comment]
> Hey @CBID2, any updates on this issue?
> 
> I'm unsure why we're converting this image into CSS if screen readers can't interpret CSS elements. My suggestion is to leave the 'Green Square' as it is and simply add the alt text 'Green Square' for accessibility.
> 
> 
> 
> CC @bdougie 

Still working on it

**Comment by nickytonline** [comment]
Unassigning you for the moment @CBID2 as the PR has been open since September 2023. If you decide to pick this up again, feel free to assign yourself.

**Comment by Arol15** [comment]
> Still exist here https://opensauced.pizza/teams
> 
> <img alt="Screen Shot 2023-07-26 at 6 59 48 AM" width="1680" src="https://private-user-images.githubusercontent.com/5713670/256255677-3453958c-c195-47cf-a302-ef515ed43883.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU4NzEwNDMsIm5iZiI6MTcwNTg3MDc0MywicGF0aCI6Ii81NzEzNjcwLzI1NjI1NTY3Ny0zNDUzOTU4Yy1jMTk1LTQ3Y2YtYTMwMi1lZjUxNWVkNDM4ODMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMjFUMjA1OTAzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTQ1NjJiYWZhNjI4ZmNjNGEzNDAwY2RhMTNkNDU1YTZhY2IwOWNjYjlmZTNiMDJmN2M2MzI4NTZhOTFjMDg3MiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Ao-sSdvuJKU2QoEEXbdLa2qZgBW1pH_mWfRQwLAK-v0">

@bdougie @a0m0rajab can you confirm that this change is only expected for the teams page and not home page? 
`GitHubMock.tsx` is not being used for the home page and making changes to it, for some reason, makes the build fail, as I think `useMediaQuery` returns null and causing type error. Or changes need to be done to `useMediaQuery` as well.

**Comment by bdougie** [comment]
Correr. That asset got moved to /teams but still exists live on the page.
